examples/package.json

```diff
@@ -7,14 +7,24 @@
     "start": "spago build && esbuild ./output/Main/index.js --bundle --serve --servedir=. --outfile=output/index.js --global-name=Main",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "Marc Cordova",
+  "author": {
+    "name": "Marc Cordova",
+    "url": "https://github.com/mcordova47"
+  },
+  "contributors": [
+    {
+      "name": "Fyodor Soikin",
+      "email": "fyodor@collegevine.com",
+      "url": "https://github.com/fsoikin"
+    }
+  ],
```

Comment:
I took the liberty of expanding the `author` field and adding myself under `contributors`.